app/style/common/variables.less

```diff
@@ -43,33 +43,38 @@
   // 6. Prepend each line with `U+`.
   // 7. Join all lines with comma.
   // 8. Prepend the list with `U+200D` (zero width joiner) and `U+FE0F` (marker telling to render glyph as emoji).
-  unicode-range: U+200d, U+FE0F, U+1f004, U+1f0cf, U+1f170-1f171, U+1f17e, U+1f17f, U+1f18e, U+1f191-1f19a,
```

Comment:
You can put `// prettier-ignore` before this line to temporarily fix the issue.

Reply:
Thanks @j-f1 totally forgot about that :)